build/Common.test.props

```diff
@@ -15,4 +20,28 @@
     <AdditionalFiles Include="$(MSBuildThisFileDirectory)/stylecop.json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+  </ItemGroup>
+
+  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Tests'))">
+    <IsTestProject>true</IsTestProject>
+    <CollectCoverage Condition="$(CollectCoverage) == ''">true</CollectCoverage>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <CollectCoverage Condition="$(CollectCoverage) == '' OR !$(TargetFramework.StartsWith('netcoreapp'))">false</CollectCoverage>
+  </PropertyGroup>
+
+  <PropertyGroup Label="CollectCodeCoverageResults" Condition="$(CollectCoverage)">
+    <CoverletOutputFormat>cobertura</CoverletOutputFormat>
+    <_BaseDir>..\..\TestResults\</_BaseDir>
+    <CoverletBaseDir>$(_BaseDir)Coverage\</CoverletBaseDir>
+    <CoverletResultsDir>$(_BaseDir)Results\</CoverletResultsDir>
+    <CoverletBaseDir Condition="$([MSBuild]::IsOSUnixLike())">$(CoverletBaseDir.Replace('\', '/'))</CoverletBaseDir>
```

Comment:
I was using `"$(OS) == 'Windows_NT'"`. Not sure what's better or faster. Perhaps this can be unified

Reply:
Which one would you like it to be unified to?

Reply:
either =) I would imagine environment variable is faster to compare. But I like descriptiveness of the method call. So I'll accept either